File: provider/src/debug/java/com/raygun/raygun4android/logging/TimberRaygunLoggerImplementation.java
package com.raygun.raygun4android.logging;

import timber.log.Timber;

public class TimberRaygunLoggerImplementation implements TimberRaygunLogger {

    public static void init() {
        if (Timber.treeCount() == 0) {
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: provider/src/debug/java/com/raygun/raygun4android/logging/TimberRaygunLoggerImplementation.kt
package com.raygun.raygun4android.logging

import timber.log.Timber
import timber.log.Timber.DebugTree

object TimberRaygunLoggerImplementation: TimberRaygunLogger {
    override fun init() {
        if (Timber.treeCount == 0) {
            Timber.plant(DebugTree())
        }
    }
}
